XenModel/Alerts/PolicyAlert.cs

```diff
@@ -141,6 +140,7 @@ public PolicyAlert(IXenConnection connection, string body)
                             sb.AppendLine();
                     }
                     Text = sb.ToString();
+                    numberOfVMsFailed = (Text == null) ? 0 : Text.Split('\n').Length;

```

Comment:
I think this could be done better here, I suggest we just count the number of VMs above (line 129)